client/components/toggle/style.scss

```diff
@@ -0,0 +1,9 @@
+.toggle__text {
+	color: $gray;
+	font-size: 11px;
+	line-height: 16px;
+	margin-left: 8px;
+	vertical-align: middle;
+	text-transform: uppercase;
+	cursor: pointer;
```

Comment:
The toggle text is not actually clickable, so it's better to actually make it clickable or remove the "hand" pointer.